src/cms/templates/language_tree/language_tree_node_form.html

```diff
@@ -41,6 +41,12 @@ <h2 class="heading font-normal">
                         </div>
                     </div>
                 </div>
+                <div class="py-2 border-b solid border-gray-200 mb-2">
+	                <div class="py-2 pb-4">
+                        {% render_field language_tree_node_form.visible|add_error_class:"border-red-500" id="visible" %}
+		                <label for="{{ language_tree_node_form.visible.id_for_label }}" class="font-bold px-2">{{ language_tree_node_form.visible.label }}</label>
+	                </div>
```

Comment:
Thanks! As I said, I think it makes sense to to add this to the `active`-field as well. I couldn't include it in the code suggestion because it was too many lines away from your changes, but since you changed the indentation now, I can suggest it in a second.